UI.NUnitVersion/Video/VideoWebUiTest.cs

```diff
@@ -69,6 +69,13 @@ protected JudgeHearingListPage LoginAsJudge(string username, string password)
         var loginPage = NavigateToVideoWeb(participant.Driver.GetDriver());
         return loginPage.LogInAsJudge(username, password);
     }
+    
+    protected PageModels.Pages.Video.Participant.StaffMemberVenueListPage LoginAsStaffMember(string username, string password)
```

Comment:
could the class definition at the bottom of the file be the reason you need the full namespace here?

Reply:
Ja